src/pages-and-resources/PagesAndResources.jsx

```diff
@@ -67,8 +67,9 @@ const PagesAndResources = ({ courseId }) => {
       <PermissionDeniedAlert />
     );
   }
-
-  const hasAdditionalCoursePlugin = getConfig()?.pluginSlots?.additional_course_plugin != null;
+  const { pluginSlots } = getConfig();
+  const hasAdditionalCoursePlugin = pluginSlots?.['org.openedx.frontend.authoring.additional_course_plugin.v1'] != null
+    && pluginSlots?.['org.openedx.frontend.authoring.additional_course_content_plugin.v1'] != null;
```

Comment:
Checking for `null` seems strange because it means that if you don't define `'org.openedx.frontend.authoring.additional_course_plugin.v1'` at all, this will be `false`, but if you define it as being empty, this will be `true`:

```jsx
const config = {
  pluginSlots: {
    'org.openedx.frontend.authoring.additional_course_content_plugin.v1': {
      plugins: [],
    },
    'org.openedx.frontend.authoring.additional_course_plugin.v1': {
      plugins: []
    }
  },
}
```